geomstats/geometry/special_orthogonal.py

```diff
@@ -1,9 +1,9 @@
 """Exposes the `SpecialOrthogonal` group class."""
 
+import geomstats.algebra_utils as utils
```

Comment:
We should try to avoid renaming the submodules, for ease of readability of the code.

Reply:
I didn't want the lines to be too long, and it doesn't really make a difference between algebra_utils and utils as 'algebra' is not even relevant in that case.